packages/compatibility/backend/sirius-components-compatibility/src/main/java/org/eclipse/sirius/components/compatibility/services/diagrams/ToolProvider.java

```diff
@@ -234,18 +236,17 @@ private SingleClickOnDiagramElementTool convertNodeCreationDescription(Map<Strin
         List<String> imagePath = this.toolImageProvider.getIcon(nodeCreationTool);
         List<IDiagramElementDescription> targetDescriptions = this.getParentNodeDescriptions(nodeCreationTool.getNodeMappings(), id2NodeDescriptions);
         var selectModelElementVariableOpt = new SelectModelElementVariableProvider().getSelectModelElementVariable(nodeCreationTool.getVariable());
-        String selectionDescriptionId = null;
-        if (selectModelElementVariableOpt.isPresent()) {
-            selectionDescriptionId = this.identifierProvider.getIdentifier(selectModelElementVariableOpt.get());
-        }
+        Dialog dialog = selectModelElementVariableOpt
+                .map(selectModelElementVariable -> new Dialog(this.identifierProvider.getIdentifier(selectModelElementVariable), SelectionDescription.TYPE))
+                .orElse(null);
         // @formatter:off
```

Comment:
Could you please remove all @formatter:off in all files you have edited, even if your code does not modify the lines were @formatter:off are?

Reply:
donne